### PR TITLE
Improve error message for failed EL translation.

### PR DESCRIPTION
**Pull request by EricEdens**
This improves error messaging for two failure modes, where the user currently sees an error such as 

1. Incorrect `--os` such that EL's translate.py gets executed. Current error message is `TranslateFailed: error: command:`.
2. Correct `--os` using BYOL, but importing a VM that doesn't have a subscription. Current error message is `TranslateFailed: error: command: Importing GPG key 0xA7317B0F`.

Testing
 - Ran:
    - daisy_integration_tests/centos_6_qcow2_translate.wf.json
    - daisy_integration_tests/centos_7_qcow2_translate.wf.json
 - Verified error message by importing a RHEL 6 image that didn't have a subscription as BYOL.
 - Successfully imported same image without using BYOL.

### CODE REVIEW

File: daisy_workflows/image_import/enterprise_linux/translate.py [hunk 1]
```diff
@@ -181,6 +177,26 @@ def DistroSpecific(g):
...
+    g (guestfs.GuestFS): A mounted GuestFS instance.
+    *packages (list of str): The YUM packages to be installed.
+
+  Raises:
+      RuntimeError: If there is a failure during installation.
+  """
+  try:
+    g.command(['yum', 'install', '-y'] + list(packages))
+  except Exception as e:
+    logging.info('Failed to install {}. Detail: {}.'.format(packages, e))
```
Comment (zoran15) [inline daisy_workflows/image_import/enterprise_linux/translate.py]: Maybe "Details"? Sounds better IMO

File: daisy_workflows/image_import/enterprise_linux/translate.py [hunk 2]
```diff
@@ -181,6 +177,26 @@ def DistroSpecific(g):
...
+    raise RuntimeError(
+        'Ensure that you have the correct --os specified. If this '
+        'is BYOL, also verify that your subscription '
+        'is eligible to be run on GCP.'.format(packages))
```
Comment (zoran15) [inline daisy_workflows/image_import/enterprise_linux/translate.py]: I don't see a placeholder for packages in the string.

File: daisy_workflows/image_import/enterprise_linux/translate.py [hunk 3]
```diff
@@ -181,6 +177,26 @@ def DistroSpecific(g):
...
+  Raises:
+      RuntimeError: If there is a failure during installation.
+  """
+  try:
+    g.command(['yum', 'install', '-y'] + list(packages))
+  except Exception as e:
+    logging.info('Failed to install {}. Detail: {}.'.format(packages, e))
+    raise RuntimeError(
+        'Ensure that you have the correct --os specified. If this '
+        'is BYOL, also verify that your subscription '
```
Comment (zoran15) [inline daisy_workflows/image_import/enterprise_linux/translate.py]: Rewording is needed for: "If this is BYOL". It's not clear what "this" is. Shawnique might help if needed.

Comment (EricEdens) [reply daisy_workflows/image_import/enterprise_linux/translate.py]: Updated to use the wording from https://cloud.google.com/compute/docs/instances/windows/bring-your-own-license/

**Pull request merged**